scalameta/scalameta/shared/src/test/scala/scala/meta/tests/prettyprinters/PublicSuite.scala

```diff
@@ -139,7 +140,7 @@ class PublicSuite extends FunSuite {
   test("scala.meta.inputs.Input.File.toString") {
     import java.io._
     import java.nio.charset.Charset
-    val path = AbsolutePath.fromRelative("hello.scala")
+    val path = RelativePath("hello.scala").absolutize(PlatformIO.workingDirectory)
```

Comment:
I think it's cleaner to have two separate concepts of `AbsolutePath` and `RelativePath`. After that, we can have converters between them.